tools/cellprofiler/common.xml

```diff
@@ -0,0 +1,445 @@
+<tool id="cp_common" name="Cellprofiler common start modules" version="@CP_VERSION@">
+    <description>
+        The first 4 modules in Cellprofiler to start a pipeline, namely Images,Metadata, NameAndTypes, Groups 
+    </description>
+    <macros>
+        <import>macros.xml</import>
+        <xml name="image_type_condition"> 
+            <conditional name="con_set_intensity">
+                <param name="set_intensity_range_from" type="select" label="Set intensity range from">
+                    <option value="Image metadata">Image metadata</option>
+                    <option value="Image bit-depth">Image bit-depth</option>
+                    <option value="Manual">Manual</option>
+                </param>
+                    <when value="Manual">
+                        <param name="maximum_intensity" type="float" value="255.0" label="Maximum intensity"/>
+                    </when>
+                    <when value="Image metadata">
+                    </when>
+                    <when value="Image bit-depth">
+                    </when>
+            </conditional>
+        </xml>
+        <xml name="name_type_rule_matching_file">
+            <param name="operator" type="select">
+                <option value="does">Does</option>
+                <option value="doesnot">Doesnot</option>

```

Comment:
please write "Does not" for the user-facing part ... UX counts :)

Reply:
I agree..But that's what appears in cp UI, but I can make the change as long as the value is the same.

Also, CP has lots of weird thing in UI, they use lots of spaces , special characters as values in its pipeline file..I hope this will be better in the next version